code/game/machinery/vending.dm

```diff
@@ -607,9 +607,15 @@
 	if (!message)
 		return
 
-	for(var/mob/O in hearers(src, null))
-		O.show_message("<span class='game say'><span class='name'>\The [src]</span> beeps, \"[message]\"</span>",2)
-	return
+	var/list/hear = hearers(src, null)
+	var/list/client/clients = list()
+	for (var/mob/M in hear)

```

Comment:
It's manually doing it so the speech bubbles work (also equivalent to `audible_message()`), and I'm not sure I want to add them to everything that calls that proc.